content/en/docs/guides/backup-and-restore.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Back up and Restore
+linkTitle: Back up and Restore
+description: "How to back up and restore resources in cozystack cluster."
+weight: 40
+aliases:
+  - /docs/backup-and-restore
+---
+
+CozyStack uses [Velero](https://velero.io/docs/v1.16/) to manage Kubernetes resource backups and recovery, including volume snapshots.
+This guide explains how to configure Velero and perform backups and recovery with practical examples.
+
+
+## Prerequisites
+
+To back up data in Cozystack, you need an S3-compatible storage ready.
+
+We recommend using the Velero CLI: [https://velero.io/docs/v1.16/basic-install/#install-the-cli](https://velero.io/docs/v1.16/basic-install/#install-the-cli)
+
+## 1. Set up Storage Credentials and Configuration
+
+To enable backups, the first step is to provide Cozystack with access to an S3-compatible storage.
+
+All configuration settings are created as secrets in the `cozy-velero` namespace.
```

Comment:
Secrets in the management cluster? Please provide an example using `kubectl create secret` or an alternative method.